tutormfe/plugin.py

```diff
@@ -107,11 +106,20 @@ def iter_mfes() -> t.Iterable[tuple[str, MFE_ATTRS_TYPE]]:
     yield from get_mfes().items()
 
 
+def iter_plugin_slots(mfe_name: str) -> t.Iterable[tuple[str, str]]:
+    """
+    Yield:
+
+        (slot_name, plugin_config)
+    """
+    yield from get_plugin_slots(mfe_name)
+
+
 def is_mfe_enabled(mfe_name: str) -> bool:
     return mfe_name in get_mfes()
 
 
-def get_mfe(mfe_name: str) -> MFE_ATTRS_TYPE:
+def get_mfe(mfe_name: str) -> t.Union[MFE_ATTRS_TYPE, t.Any]:
```

Comment:
@hinakhadim, your previous comment got me on the right track, thanks - but I didn't want to change running code just for the sake of a type hint, so this is what I did.  It seems there's significant precedent for `Any` in the codebase, so I expect it to be fine.

Reply:
The issue is caused by the lru_cache decorator, which is not correctly typed. You can bypass this issue without `Any` by writing:

```
def get_mfe(mfe_name: str) -> MFE_ATTRS_TYPE:
    mfe: MFE_ATTRS_TYPE = get_mfes().get(mfe_name, {})
    return mfe
```